### ISSUE TITLE
Gastric acid reducing therapies improve symptoms

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10281]
**Preferred term label:**
Gastric acid reducing therapies improve symptoms

**Synonyms**
PPI; H2B; proton pump inhibitor; histamine receptor blocker; omeprazole; esomeprazole; dexlansoprazole; pantoprazole; lansoprazole; famotidine; ranitidine

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
GERD but not so much eosinophilic esophagitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
This applies only to one symptom, which is not currently in HPO
I think it would be better to have
Heartburn: A painful burning sensation in the chest or upper central abdomen.
Comment: Heartburn is typically caused by gastroesophageal reflux disease, is a chronic gastrointestinal disorder characterized by the regurgitation of gastric contents into the esophagus. The mechanism is related to the disruption of the esophagogastric junction barrier resulting in exposure of the esophagus to acidic gastric contents.
Syn: Pyrosis, Cardialgia, Acid indigestion
PMID: 28722967

And then have Gastric acid reducing therapy refractory heartburn
thoughts?

Currently, Heartburn is listed as a RELATED_SYNONYM to GERD, note to me -do not forget to remove this.

**Comment by MickeySegal** [comment]
The effect of reducing acid is a property of many different pathologies besides GERD, e.g., duodenal ulcer and eosinophilic esophagitis.  The value of this finding is that it is a question you can ask and get information that is useful diagnostically.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000350
- label: Ameliorated by gastric acid-reducing therapy
- parents: [Ameliorated by exposure to medication (HP:0034536)](https://hpo.jax.org/app/browse/term/HP:0034536)
- pmid: 
- def: Applies to a sign or symptom that is improved or made more bearable by treatment with an agent that reduces gastric acidity such as proton pump inhibitors, histamine receptor blockers,omeprazole, esomeprazole, dexlansoprazole, pantoprazole, lansoprazole, famotidine, or ranitidine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10281
- synonyms:
